emcc.py

```diff
@@ -2821,6 +2821,11 @@ def check_bad_eq(arg):
           newargs[i] = '-g'
           shared.Settings.FULL_DWARF = 1
           shared.warning('gforce_dwarf is a temporary option that will eventually disappear')
+        elif requested_level.startswith('side'):
```

Comment:
Does this still suffer from the problem that -g after -gside negates the SIDE_DEBUG setting?
Actually looking at the code, I don't really see how that happens. FULL_DWARF will get set to 1, but then nothing unsets it when the -g is later processed?

Reply:
Yeah, that flag mixup was something else I think. @pfaffe let me know if that's still an issue and if I can help there.

Overall `-g` can't override the two new flags we added, so this should be ok.